docs/openapi.yml

```diff
@@ -245,6 +246,80 @@ paths:
         500:
           description: Any other error
 
+  /api/v1/projects/{id}:/bind/end:
+    post:
+      summary: Complete binding the project to Codewind
+      description: >
+        Complete binding the project to codewind. The project will be enabled and built.
+      parameters:
+        - name: id
+          in: path
+          schema:
+            $ref: '#/components/schemas/ProjectID'
+          required: true
+          description: id of the project        
+      responses:
+        202:
+          description: >
+            Bind end request accepted. Will attempt to complete binding the Codewind project.
+        404:
+          description: Project with given projectiD was not found.
```

Comment:
`projectiD` - I think you've managed to find a formatting of projectId we haven't used before. ;-)